services/storage/src/simcore_service_storage/s3_client.py

```diff
@@ -298,11 +308,16 @@ async def delete_files_in_project_node(
     async def get_file_metadata(
         self, bucket: S3BucketName, file_id: SimcoreS3FileID
     ) -> S3MetaData:
-        response = await self.client.head_object(Bucket=bucket, Key=file_id)
+        response = await self.client.head_object(
+            Bucket=bucket, Key=file_id, ChecksumMode="ENABLED"
+        )
         return S3MetaData(
             file_id=file_id,
             last_modified=response["LastModified"],
             e_tag=json.loads(response["ETag"]),
+            sha256_checksum=SHA256Str(response.get("ChecksumSHA256"))
```

Comment:
I think doing `SHA256Str(response.get("ChecksumSHA256"))` will not validate the result as SHA and at the same time `S3MetaData` will think it is already validated.  I would just
```python
S3MetaData(..., sha256_checksum=response.get("ChecksumSHA256") )
```
I am aware that pylance is complain but the pydantic class will parse and validate that input from a string but pytlance does not know that i.e. it is a false positive

Reply:
done